cmd/syncthing/cli/show.go

```diff
@@ -35,6 +35,11 @@ var showCommand = cli.Command{
 			Usage:  "Report about connections to other devices",
 			Action: expects(0, indexDumpOutput("system/connections")),
 		},
+		{
+			Name:   "discovery",
+			Usage:  "Report about the local discovery cache",
+			Action: expects(0, indexDumpOutput("system/discovery")),
```

Comment:
That's definitely a misnomer by now - `prettyPrintOutput` or something would be more appropriate. Not something needed to be addressed here.